docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md

```diff
@@ -254,6 +254,12 @@ The Alert object represents an alert included in the notification group, as prov
 
 ## Custom Payload
 
+{{< admonition type="note" >}}
+
+Custom Payload is not [generally available](https://grafana.com/docs/release-life-cycle/#general-availability) by default in Grafana Cloud.
```

Comment:
It will be available by default once it rolls out this week, so the "by default" is misleading IMO. Maybe something like:
```suggestion
Custom Payload is not yet [generally available](https://grafana.com/docs/release-life-cycle/#general-availability) in Grafana Cloud.
```

Reply:
addressed by https://github.com/grafana/grafana/pull/104331/commits/f438e0ded3feffbf115b3d30886c633bb7f0b5cd